src/lib/dracut/modules.d/30gcp-udev-rules/module-setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/bash
+# Install 64-gce-disk-removal.rules, 65-gce-disk-naming.rules and
+# google_nvme_id into the initramfs
+
+# called by dracut
+install() {
+    inst_simple /usr/lib/udev/google_nvme_id
+    inst_multiple \
+        /usr/lib/udev/rules.d/64-gce-disk-removal.rules \
```

Comment:
Actually, since you are looking to use this dracut module- why not remove it entirely?

Reply:
I'm not sure I quite understand 😀

We can't merge #51 yet because some distros might be relying on it (still under investigation), but at least in the Fedora RPM (and probably RHEL) we'll just add a patch to remove the file.

This dracut module is net new and all it does is add these files to the initramfs so the rules will be evaluated early enough that initramfs processes can rely on them (we run Ignition from the initramfs and can do partitioning/filesystem creation, etc, there). Since none of the older systems that rely on the `64-gce-disk-removal.rules` will be picking up and using this dracut module (I hope) then we can just not copy in `64-gce-disk-removal.rules` into the initramfs by not specifying the line here.

My line of reasoning may be flawed. Let me know if I'm missing anything.

Reply:
Exactly- I mean don't include `64-gce-disk-removal.rules` in this change.

Reply:
Thanks all for the comments ;)